src/Components/Header.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import logo from "../asset/logo.jpeg";
+import user from "../asset/user.png";
+
+const HeaderComponent = () => {
+  return (
+    <div className="header">
+      <Logo></Logo>
+      <SearchBar></SearchBar>
```

Comment:
It will be good to keep search bar outside header container.
That way if some other page doesn't contains search bar we wont not have to rewrite our code.
Header and footer should have static data. :)